cypress/integration/metaNewsSpec.js

```diff
@@ -98,14 +99,22 @@ describe('Article Meta Tests', () => {
     metadataAssertion();
   });
 
-  it('should include the canonical URL', () => {
+  it('should include the canonical URL & ampHTML', () => {
+    const currentOrigin = window.location.origin;
     checkCanonicalURL(
-      `https://www.bbc.com/news/articles/${
+      `${currentOrigin}/news/articles/${config.assets.newsThreeSubheadlines}`,
+    );
+    checkAmpHTML(
+      `${currentOrigin}/news/articles/${
         config.assets.newsThreeSubheadlines
-      }`,
+      }.amp`,
     );
   });
 
+  it('should include metadata that matches the JSON data', () => {
+    metadataAssertion();
```

Comment:
Is the metadata change meant to be part of this PR?

Reply:
Good point - a duplicate test snuck in. Removed it now